### PR TITLE
Document method of retriggering post-merge builds from the GitHub UI.

### PR DESCRIPTION
**Pull request by ralphbean**

### CODE REVIEW

**Review by arewm — APPROVED** [review]

File: docs/modules/ROOT/pages/how-to-guides/configuring-builds/proc_retriggering_build_pipelines.adoc [hunk 1]
```diff
@@ -18,6 +18,21 @@ You should see the pipelinerun start executing in the UI and in the pull request
...
+== Retriggering a Post-merge build from your main branch from GitHub
+
+.**Prerequisite**
+
+- You have already merged a pull request, but the subsequent build failed, prompting a need for retriggering.
+
+.**Procedure**
+
+. Navigate to the latest commit on your branch in the GitHub UI.
+. Comment `/test` on the commit.
```
Comment (savitaashture) [inline docs/modules/ROOT/pages/how-to-guides/configuring-builds/proc_retriggering_build_pipelines.adoc:30]:
```suggestion
. Comment 
* `/test` , `/test <pipelinerun_name>`
* `/retest` , `/retest <pipelinerun_name>`  on the commit.
```

File: docs/modules/ROOT/pages/how-to-guides/configuring-builds/proc_retriggering_build_pipelines.adoc [hunk 2]
```diff
@@ -18,6 +18,23 @@ You should see the pipelinerun start executing in the UI and in the pull request
...
 
+== Retriggering a Post-merge build from your main branch from GitHub
+
+.**Prerequisite**
+
+- You have already merged a pull request, but the subsequent build failed, prompting a need for retriggering.
+
+.**Procedure**
+
+. Navigate to the latest commit on your branch in the GitHub UI.
```
Comment (gtrivedi88) [inline docs/modules/ROOT/pages/how-to-guides/configuring-builds/proc_retriggering_build_pipelines.adoc:29]: On the GitHub UI, navigate to the latest commit on your branch.

File: docs/modules/ROOT/pages/how-to-guides/configuring-builds/proc_retriggering_build_pipelines.adoc [hunk 3]
```diff
@@ -18,6 +18,23 @@ You should see the pipelinerun start executing in the UI and in the pull request
...
+. Comment
```
Comment (gtrivedi88) [inline docs/modules/ROOT/pages/how-to-guides/configuring-builds/proc_retriggering_build_pipelines.adoc:30]: Do we want to say = Add the following comments:?

File: docs/modules/ROOT/pages/how-to-guides/configuring-builds/proc_retriggering_build_pipelines.adoc [hunk 4]
```diff
@@ -18,6 +18,23 @@ You should see the pipelinerun start executing in the UI and in the pull request
...
+- You have already merged a pull request, but the subsequent build failed, prompting a need for retriggering.
+
+.**Procedure**
+
+. Navigate to the latest commit on your branch in the GitHub UI.
+. Comment 
+* `/test` , `/test <pipelinerun_name>`
+* `/retest` , `/retest <pipelinerun_name>`  on the commit.
+
+The pipeline run should resume in the *Activity* > *Pipeline runs* tab.
```
Comment (gtrivedi88) [inline docs/modules/ROOT/pages/how-to-guides/configuring-builds/proc_retriggering_build_pipelines.adoc:34]:
```suggestion
+
The pipeline run should resume in the *Activity* > *Pipeline runs* tab.
```

File: docs/modules/ROOT/pages/how-to-guides/configuring-builds/proc_retriggering_build_pipelines.adoc [hunk 5]
```diff
@@ -18,6 +18,23 @@ You should see the pipelinerun start executing in the UI and in the pull request
...
+
+NOTE: This only works on the latest commit on the branch.
```
Comment (gtrivedi88) [inline docs/modules/ROOT/pages/how-to-guides/configuring-builds/proc_retriggering_build_pipelines.adoc:36]:
```suggestion
+
NOTE: This only works on the latest commit on the branch.
```

**Review by gtrivedi88 — APPROVED** [review]

**Comment by dirgim** [comment]
Just a note that this page isn't currently visible in the [documentation navigation](https://redhat-appstudio.github.io/docs.appstudio.io/Documentation/main/). I'm guessing that [this navpage](https://github.com/redhat-appstudio/docs.appstudio.io/blob/main/docs/modules/ROOT/nav-how-to-guides.adoc) needs to be updated too.

**Comment by arewm** [comment]
The redhat-appstudio docs are deprecated. Would you be able to recreate a similar PR on [konflux-ci/docs](https://github.com/konflux-ci/docs)?

**Comment by arewm** [comment]
I have migrated this PR to the new docs: https://github.com/konflux-ci/docs/pull/100

**Pull request closed without merging**